resources/js/components/status.js

```diff
@@ -0,0 +1,12 @@
+let currentDate = '';
+
+Array.from(document.getElementsByClassName('status')).forEach(elem => {
+   if (elem.dataset.date != currentDate) {
+       currentDate = elem.dataset.date;
+       const heading = document.createElement("h5");
+       const textnode = document.createTextNode(currentDate);
+       heading.classList.add("mt-4");
+       heading.appendChild(textnode);
+       elem.parentNode.insertBefore(heading, elem);
+   }
+});
```

Comment:
Warum soherum? Das JS verschiebt die komplette Seite, wenn die Headings eingefügt werden. Fand die alte PHP-Version schicker, abgesehen davon, dass sie funktioniert, wenn JS ausgeschaltet sein sollte (oder später/nicht geladen werden kann).

Reply:
Das Problem ist, dass php-Tags und Variablenzuweisungen in Blade relativ inperformant sind und Blade auch davon abrät, PHP-Tags zu nutzen. Außerdem soll ja Träwelling auf lange sicht eh mit vue gemacht werden, also ist Javascript eh verpflichtend. :/

Reply:
Mhkey :/